spec/index.bs

```diff
@@ -2737,10 +2764,19 @@ interface GPUAdapter {
 
     : <dfn>requestAdapterInfo()</dfn>
     ::
-        Requests the {{GPUAdapterInfo}} for this {{GPUAdapter}}.
+        Requests {{GPUAdapterInfo}} for this {{GPUAdapter}}.
 
-        Note: Adapter info values are returned with a Promise to give user agents an
-        opportunity to perform potentially long-running checks in the future.
+        Returns a new object each time, whereas <code>adapter.{{GPUAdapter/info}}</code>
+        returns the same object each time.
+
+        <div class=note heading>
+            This is deprecated (but not planned for removal) in favor of the newer, synchronous
```

Comment:
Do we want to indicate that this isn't planned for removal? Obviously we won't be stripping it out at the moment, but I think that long term we'd like to lean towards removing this. In fact, if Mozilla and Apple are given leeway to not ship this then it should be relatively easy for Chrome to phase it out.